jekyll/_cci2/deployment-integrations.md

```diff
@@ -142,137 +142,44 @@ To deploy to Azure, use a similar job to the above example that uses an appropri
+create a `deploy-job`,
+`checkout` your code,
+and add a command
+to deploy the master branch to Heroku via git.
 
 ```yaml
 version: 2
 jobs:
-  # jobs for building/testing go here
+  build-job:
+    ...
   deploy-job:
-    docker:
-      - image: my-image
-    working_directory: /tmp/my-project
     steps:
       - checkout
-      - add_ssh_keys:  # add key from CircleCI account based on fingerprint
-          fingerprints:
-            - "48:a0:87:54:ca:75:32:12:c6:9e:a2:77:a4:7a:08:a4"
-      - run:
-          name: Run Setup Script
-          command: bash .circleci/setup-heroku.sh
       - run:
           name: Deploy Master to Heroku
-          command: |  # this command is framework-dependent and may vary
-            heroku git:remote -a $HEROKU_APP_NAME
-            git push --force git@heroku.com:$HEROKU_APP_NAME.git HEAD:refs/heads/master
-            heroku run python manage.py deploy
-            heroku restart
-workflows:
-  version: 2
-  build-deploy:
-    jobs:
-      - build-job
-      - deploy-jobs:  # only deploy when master successfully builds
-          requires:
-            - build-job
-          filters:
-            branches:
-              only: master
-```
-
-If you prefer to deploy to Heroku using https, please consider this circleci (2.0) configuration which is even easier:
-
-Since we are deploying our app with https, the bash script above needed to setup heroku with ssh is not required anymore.
-For that reason the following steps can be deleted from the configuration:
-
-```yaml
-- add_ssh_keys:  # add key from CircleCI account based on fingerprint
-            fingerprints:
-            - "48:a0:87:54:ca:75:32:12:c6:9e:a2:77:a4:7a:08:a4"
-- run:
-     name: Run Setup Script
-     command: bash .circleci/setup-heroku.sh
-```
-
-instead we use the following step
-
-```yaml
-- run:
-     name: Deploy to Heroku
-        command: |
-            git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git HEAD:refs/heads/master
-            sleep 5  # sleep for 5 seconds to wait for dynos
-            heroku restart
-```
-
-- Result:
+          command: |
+            git push https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git master
 
-```yaml
-version: 2
-jobs:
-  # jobs for building/testing go here
-  deploy-job:
-    docker:
-      - image: my-image
-    working_directory: /tmp/my-project
-    steps:
-      - checkout
-      - run:
-          name: Deploy Master to Heroku
-          command: |  # this command is framework-dependent and may vary
-            heroku git:remote -a HEROKU_APP_NAME
-            git push --force https://heroku:$HEROKU_API_KEY@git.heroku.com/$HEROKU_APP_NAME.git HEAD:refs/heads/master
-            sleep 5  # sleep for 5 seconds to wait for dynos
-            heroku restart
 workflows:
   version: 2
   build-deploy:
     jobs:
       - build-job
-      - deploy-jobs:  # only deploy when master successfully builds
+      - deploy-job:
```

Comment:
In the next section, we should change 'for additional details' or remove that link entirely (and if that documentation is incorrect/not what we should tell people, we should remove it from the python flask demo app as well and replace it with the easy way). All of this info being removed is still in the Project Walkthrough.

Reply:
Awesome. I'll remove this link for now and make a ticket to update the Project Walkthrough with the easier Heroku deployment method.